src/ex1.py

```diff
@@ -0,0 +1,28 @@
+#Improve this program using a switch-case. 
+week=int(input("Insert a number from 1 to 7: "))
```

Comment:
potresti mettere questo dentro `if __name__ == "__main__" https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do
e mettere dentro una funzione il risultato di "match week.."

una funzione che prende in input `week` e restituisca in output la stringa `it's monday` (o le altre)

esempio:
```python
def get_day_of_week(week: int) -> str:
....

if __name__ == "__main__":
    week=int(input("Insert a number from 1 to 7: "))
    day = get_day_of_week(week)
    print(day)
```